README.md

```diff
@@ -5,22 +5,22 @@ Here you can read and add song lyrics.
 
 ### What this project is contributing to the world
 
- - **problem**: quality of lyrics on the Internet leaves something to be desired, often following the basic quantity over quality paradigm  
+ - **problem**: quality of lyrics on the Internet leaves something to be desired, often following the basic quantity over quality paradigm
  **solution**: maintain lyrics database as an open source project, let people contribute texts to songs they love; reach out to original authors, kindly asking to share their lyrics with the world
 
- - **problem**: little motivation contributing to resources that may disappear  
+ - **problem**: little motivation contributing to resources that may disappear
  **solution**: to never lose the progress, make it possible for anyone to easily download the whole database
 
- - **problem**: lyrics websites are often unpleasant to view due to inaccessible design and large amount of ads  
+ - **problem**: lyrics websites are often unpleasant to view due to inaccessible design and large amount of ads
  **solution**: create and launch a lightweight open-source lyrics website
 
- - **problem**: it's generally not easy to maintain quality control over large datasets  
+ - **problem**: it's generally not easy to maintain quality control over large datasets
```

Comment:
I think the double spaces need to stay, in Markdown it indicates a forced newline... those problem/solution pairs are meant to be one on top of the other

Reply:
My editor is set to automatically remove trailing whitespace on all documents. I don't think it should ever be required in markdown docs. But I'll try to reformat that the way its intended.

Reply:
I read up on line breaks at https://github.github.com/gfm/

My last commit puts a single backslash at the end, with no trailing whitespace

Preview looks like this:

![image](https://user-images.githubusercontent.com/16764864/102663529-a794e000-4146-11eb-8eb1-6ff8a395ed17.png)

We good?

Reply:
If you're ok with the backslash, let me know and I'll go ahead and use that in the other markdown docs where it's needed to match what's currently in the wiki.

Reply:
Oh, didn't know about a backslash, seems like a much better way of adding newlines to Markdown files!

Reference link: https://github.github.com/gfm/#example-655